lib/resources/r4/charge_item_modify.yaml

```diff
@@ -0,0 +1,559 @@
+                      "valueUnsignedInt": 1,
+                      "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/priority"
+                    }
+                  ],
+                  "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/procedure-code"
+                }
+              ]
+            }
+          }
+        ]
+      }
+    note: >
+      <ul>
+        <li>The coding field must contain only one coding from the CPT, HCPCS, or PROCCODE proprietary Bill Codes Types systems</li>
+        <li>Must be of a lower bill code tier than the ChargeItem.code field if provided.</li>
+        <li>It Must have priority extension</li>
+        <li>The value of priority must be in the form of `valueUnsignedInt`</li>
+      </ul>
+    url: http://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.extension
+
+  - name: Revenue Code Extension
+    required: 'No'
+    type: Extension
+    description: The type of revenue or cost center providing the product and/or service.
+    action: modify
+    example: |
+      {
+        "parameter": [
+          {
+            "resource": {
+              "extension": [
+                {
+                  "valueCodeableConcept": {
+                    "coding": [
+                      {
+                        "extension": [
+                          {
+                            "valueCoding": {
+                              "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/14002",
+                              "code": "615217"
+                            },
+                            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/bill-code-schedule"
+                          }
+                        ],
+                        "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/CodeSystem/BillCodes-REVENUE",
+                        "code": "0024"
+                      }
+                    ]
+                  },
+                  "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/revenue-code"
+                }
+              ]
+            }
+          }
+        ]
+      }
+    note: >
+      <ul>
+        <li>The coding field must contain only one coding from the REVENUE proprietary Bill Codes Types system</li>
+      </ul>
+    url: http://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.extension
+
+  - name: transactionId
+    required: 'Yes'
+    type: uuid
+    description: A UUID to uniquely identify the credit operation.
+    action: modify
+    example: |
+      {
+        "parameter": [
+          {
+            "name": "transactionId",
+            "valueUuid": "urn:uuid:aa7623a0-0c7b-11ec-82a8-0242ac130003"
+          }
+        ]
+      }
+    note: >
+      <ul>
+        <li>The value must be in the form of `valueUuid`</li>
+        <li>Only one value can be defined</li>
+      </ul>
+    url: http://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.transactionId
```

Comment:
This link doesn't actually exist, I think we would just want to link to http://hl7.org/fhir/r4/parameters-definitions.html#Parameters.parameter here.

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/780/commits/2e95e8f7f7c9760a3c5d4d4f5df05fb503cb5543#diff-3c565d9c41e2bb3d417efe63a774683e72b07cd38f2ff54a695c695999f29938R559